public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>TeamUp</title>
```

Comment:
what about the application img ? 
![image](https://user-images.githubusercontent.com/85913826/130685413-bcc6a07c-26b1-4030-a3cc-3eb997893ab1.png)

Reply:
The app image is up